yarn-project/circuits.js/src/structs/kernel/__snapshots__/index.test.ts.snap

```diff
@@ -312,133 +312,320 @@ chain_id: 0x501
+portal_contract_address: 0x4
+
+
+is_execution_request: 0
+ contract_address: 0x1013
+function_data: function_selector: 1014
+is_private: 1
+is_constructor: 1
+
+public_inputs: call_context: msg_sender: 0x1024
+storage_contract_address: 0x1025
+portal_contract_address: 0x1026
+is_delegate_call: 1
+is_static_call: 1
+is_contract_deployment: 1
+
+args: [ 0x1123 0x1124 0x1125 0x1126 0x1127 0x1128 0x1129 0x112a ]
+return_values: [ 0x1323 0x1324 0x1325 0x1326 ]
+emitted_events: [ 0x1223 0x1224 0x1225 0x1226 ]
+new_commitments: [ 0x1423 0x1424 0x1425 0x1426 ]
+new_nullifiers: [ 0x1523 0x1524 0x1525 0x1526 ]
+private_call_stack: [ 0x1623 0x1624 0x1625 0x1626 ]
+public_call_stack: [ 0x1723 0x1724 0x1725 0x1726 ]
+new_l2_to_l1_msgs: [ 0x1823 0x1824 ]
+historic_private_data_tree_root: 0x2023
+historic_nullifier_tree_root: 0x2123
+historic_contract_tree_root: 0x1923
+historic_l1_to_l2_messages_tree_root: 0x2223
+contract_deployment_data: constructor_vk_hash: 0x1
+function_tree_root: 0x2
+contract_address_salt: 0x3
+portal_contract_address: 0x4
+
+
+is_execution_request: 0
+ contract_address: 0x1014
+function_data: function_selector: 1015
+is_private: 1
+is_constructor: 1
+
+public_inputs: call_context: msg_sender: 0x1025
+storage_contract_address: 0x1026
+portal_contract_address: 0x1027
+is_delegate_call: 1
+is_static_call: 1
+is_contract_deployment: 1
+
+args: [ 0x1124 0x1125 0x1126 0x1127 0x1128 0x1129 0x112a 0x112b ]
+return_values: [ 0x1324 0x1325 0x1326 0x1327 ]
+emitted_events: [ 0x1224 0x1225 0x1226 0x1227 ]
+new_commitments: [ 0x1424 0x1425 0x1426 0x1427 ]
+new_nullifiers: [ 0x1524 0x1525 0x1526 0x1527 ]
+private_call_stack: [ 0x1624 0x1625 0x1626 0x1627 ]
+public_call_stack: [ 0x1724 0x1725 0x1726 0x1727 ]
+new_l2_to_l1_msgs: [ 0x1824 0x1825 ]
+historic_private_data_tree_root: 0x2024
+historic_nullifier_tree_root: 0x2124
+historic_contract_tree_root: 0x1924
+historic_l1_to_l2_messages_tree_root: 0x2224
+contract_deployment_data: constructor_vk_hash: 0x1
+function_tree_root: 0x2
+contract_address_salt: 0x3
+portal_contract_address: 0x4
+
+
+is_execution_request: 0
+ ]
+proof:
+[ 51 51 51 51 51 51 51 51 51 51 51 51 51 51 51 51 ]
+vk:
+key.composer_type: 0
+key.circuit_size: 101
+key.num_public_inputs: 102
+key.commitments: [
+ A: { 0x200, 0x300 }
+]
+key.contains_recursive_proof: 1
+key.recursive_proof_public_input_indices: [ 400 401 402 403 404 ]
+
+function_leaf_membership_witness:
+leaf_index: 0x1031
+sibling_path: [ 0x1031 0x1032 0x1033 0x1034 ]
+
+contract_leaf_membership_witness:
+leaf_index: 0x1021
+sibling_path: [ 0x1021 0x1022 0x1023 0x1024 ]
+
+portal_contract_address: 0x4141414141414141414141414141414141414141
+acir_hash: 0x1061
+
+"
+`;
+
+exports[`structs/kernel serializes and prints private_kernel_inputs_inner 1`] = `
+"previous_kernel:
 public_inputs: end:
 aggregation_object:
-P0: { 0x1001, 0x1002 }
-P1: { 0x1101, 0x1102 }
+P0: { 0x1, 0x2 }
+P1: { 0x101, 0x102 }
 public_inputs: [
- 0x1003
- 0x1004
- 0x1005
- 0x1006
+ 0x3
```

Comment:
These should have be unchanged, no?

Reply:
The seed to creating a "fake" `PreviousKernelData` and `PrivateCallData` has been changed slightly (because of introduction of `PrivateKernelInputsInit` and `PrivateKernelInputsInner`). So its okay that the snapshot has changed.